src/components/TemplateSubmissionForm.tsx

```diff
@@ -73,11 +71,6 @@ const TemplateSubmissionForm: React.FC<TemplateSubmissionFormProps> = ({
       <Typography variant="body1" align="center">
         Maintainer: {maintainer}
       </Typography>
-      {repository && (
```

Comment:
Graph-nightly should be up to date now with Workflows main, so repository can go back in (if you want it).